stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -212,6 +204,13 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                 }
             }
         }
+
+        supportFragmentManager.executePendingTransactions()
```

Comment:
Can you add a comment on why you used executePendingTransactions and doOnNextLayout.  This function call continues to come and go :D

Reply:
Done